src/tools/optimization-options.h

```diff
@@ -165,6 +165,13 @@ struct OptimizationOptions : public ToolOptions {
              passOptions.inlining.oneCallerInlineMaxSize =
                static_cast<Index>(atoi(argument.c_str()));
            })
+      .add("--allow-heavyweight-functions",
```

Comment:
```suggestion
      .add("--inline-heavyweight-functions",
```
like the other inlining flags I think this should mention inlining somehow - otherwise the name may be confusing.